sunbird-cassandra-migration/cassandra-migration/src/main/resources/db/migration/cassandra/V1.107_cassandra.cql

```diff
@@ -0,0 +1,12 @@
+
+CREATE KEYSPACE IF NOT EXISTS sunbird_groups WITH replication = {'class':'SimpleStrategy','replication_factor':3};
```

Comment:
Dont provide SimpleStratgey, provide NetworkTopology instead.

Reply:
Discussed with @SanthoshVasabhaktula. Okay to go with simplestrategy.